utils/isMaximumStorageLimitReached.ts

```diff
@@ -0,0 +1,19 @@
+import { Platform } from 'react-native';
+import {
+  getFreeDiskStorageOldSync,
+  getFreeDiskStorageSync,
+} from 'react-native-device-info';
+import getAllConfigurations from '../shared/commonprops/commonProps';
+
+export default async function isMaximumStorageLimitReached() {
```

Comment:
Can we revisit the file name? "isMaximumStorageLimitReached" looks file for a function name but not a file name.

Reply:
This refactoring can be taken as a part of ongoing story INJI-172